content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,112 @@
+
+Istio 1.3 adds experimental support to generate service-level HTTP metrics
+directly in the Envoy proxies. This feature lets you continue to monitor your service meshes using the tools Istio provides
+without needing Mixer. This support is the first step towards the new
+extensibility architecture for Istio. This architecture puts Mixer-like functionality directly in the Envoy proxies.
+
+The in-proxy generation of service-level metrics replaces the following HTTP metrics that Mixer currently
+generates:
+
+- `istio_requests_total`
+- `istio_request_duration_*`
+- `istio_request_size`
+
+## Enable Service-level Metrics Generation in Envoy
+
+To generate service-level metrics directly in the Envoy proxies, follow these steps:
+
+1.  To prevent duplicate telemetry generation, turn off the `istio-telemetry` service to disable calls to `istio-telemetry` in the mesh:
+
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.telemetry.enabled=false --set mixer.policy.enabled=false
+    {{< /text >}}
+
+        {{< tip >}}
+        Alternatively, you can comment out `mixerCheckServer` and `mixerReportServer` in your mesh's configuration file.
+        {{< /tip >}}
+
+1. To generate service-level metrics, the proxies must exchange {{< gloss >}}workload{{< /gloss >}} metadata. 
+   A custom filter handles this exchange. Enable the metadata exchange filter with the following command:
+
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
+    {{< /text >}}
+
+1. To actually generate the service-level metrics, you must apply the custom stats filter:
+
+    A custom filter must also be applied to actually generate the service-level metrics.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+    {{< /text >}}
+
+1. Go to the **Istio Mesh** dashboard Grafana. Verify that the dashboard displays the same telemetry as before but without
+    any requests flowing through Istio's Mixer.
+
+
+## Differences with Mixer-based generation
+
+Small differences between the in-proxy generation and Mixer-based generation of service-level metrics
+persist in Istio 1.3. We won't consider the functionality stable until in-proxy generation has full feature-parity with
+Mixer-based generation.
+
+Until then, please consider these differences:
+
+- The `istio_request_duration_seconds` latency metric has the new name: `istio_request_duration_milliseconds`. 
+  The new metric uses milliseconds instead of seconds. We updated the Grafana dashboards to
+  account for these changes.
+- The `istio_request_duration_milliseconds` metric uses more granular buckets inside the proxy.
+ This change leads to lower latency measurements in the histograms than previously observed.
```

Comment:
I'm not clear on why more histogram buckets results in different latency observations. Please clarify

Reply:
reworded and avoided "lower" comparison.